cmd/commands/runtime.go

```diff
@@ -187,7 +187,10 @@ func NewRuntimeInstallCommand() *cobra.Command {
 			err := runtimeInstallCommandPreRunHandler(cmd, &installationOpts)
 			handleCliStep(reporter.InstallPhasePreCheckFinish, "Finished pre installation checks", err, false)
 			if err != nil {
-				return util.DecorateErrorWithDocsLink(fmt.Errorf("Pre installation error: %w", err), store.Get().RequirementsLink)
+				if err.Error() != "^C" {
```

Comment:
Is there a better way to recognize this? pretty sure it would be different between different operating system. Does the error have a type?